de._0x539.gobby.json

```diff
@@ -0,0 +1,181 @@
+{
+    "app-id": "de._0x539.gobby",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "43",
+    "sdk": "org.gnome.Sdk",
+    "command": "gobby-0.5",
```

Comment:
but the source pull 0.6.0

Reply:
That's what the executable is called! I think because 0.5 was backwards-incompatible with the previous versions' wire protocol.

Reply:
it felt weird.